examples/siro_sandbox/sandbox_app.py

```diff
@@ -170,12 +172,57 @@ def __init__(self, args, config, gui_input):
         self._cursor_style = None
         self._can_grasp_place_threshold = args.can_grasp_place_threshold
 
+        self._num_iter_episodes: int = len(self.env.episode_iterator.episodes)  # type: ignore
+        self._num_episodes_done: int = 0
         self._reset_environment()
 
+    def _make_dataset(self, config):
+        from habitat.datasets import make_dataset
+
+        dataset_config = config.habitat.dataset
+        dataset = make_dataset(
+            id_dataset=dataset_config.type, config=dataset_config
+        )
+
+        if self._play_episodes_filter_str is not None:
+            _max_num_digits: int = len(str(len(dataset.episodes)))
```

Comment:
nit: don't need underscore for local var

Reply:
Done!